plasmapy/physics/transport.py

```diff
@@ -2125,7 +2125,7 @@ def _nondim_tc_e_ji_held(hall, Z, field_orientation):
 
     allowed_Z = [1, 2, 'arbitrary']
     Z_idx = _check_Z(allowed_Z, Z)
-    r = np.abs(Z * hall)
+    r = float(np.abs(Z * hall))
```

Comment:
whaaaaaaaaaaaat. :laughing:

Reply:
It's cuz `Z` and `hall` are both type `int` in the tests so when you cubed it it still tried to keep it as `int` and it overflowed! 😭